lib/datadog/core/environment/ext.rb

```diff
@@ -33,8 +33,14 @@ module Ext
         LANG_INTERPRETER = "#{RUBY_ENGINE}-#{RUBY_PLATFORM}"
         LANG_PLATFORM = RUBY_PLATFORM
         LANG_VERSION = RUBY_VERSION
+        PROCESS_TYPE = 'script'
```

Comment:
We both know why this is `script`, but can let's add a comment here about what script means, so we remember in the future why Ruby is always a script process type today.